plugin/aws/profile.go

```diff
@@ -29,7 +29,7 @@ func newProfile(ctx context.Context, name string) (*profile, error) {
 
 	// profile-specific stdin prompt
 	tokenProvider := func() (string, error) {
-		return plugin.Prompt(fmt.Sprintf("Assume ROLE MFA token code for %v", name))
+		return plugin.Prompt(fmt.Sprintf("Enter MFA token code for role %v", name))
```

Comment:
Hmm, actually the name is of the profile. Maybe we need to mention role.

Reply:
Updated again

Reply:
Ugh, I meant to say "don't need to", but what's here seems fine.

Reply:
No worries. Updated again.